FrameworkSystem/DB/InstalledComponentsDB.py

```diff
@@ -244,7 +244,7 @@ class HostLogging( componentsBase ):
   hostName = Column( 'HostName', String( 32 ), nullable = False, primary_key = True )
   # status
   DIRAC = Column( 'DIRACVersion', String( 32 ) )
-  Extension = Column( 'Extension', String( 64 ) )
+  Extensions = Column( 'Extensions', String( 64 ) )
```

Comment:
Yes. I do not like having the same things 10 different names. We use Extensions in DIRAC we should not invent a new name.